subsys/dsp/cmsis_backend/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+target_include_directories(zdsp PUBLIC public_overrides)
+
+zephyr_compile_definitions_ifndef(CONFIG_ARM __GNUC_PYTHON__)
```

Comment:
This really needs to be in the CMSIS-DSP integration code itself, not here in "DSP subsystem."